lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -46,6 +47,7 @@
 uiucedu_is_member_of = "uiucedu_is_member_of"
 DEBUG_ON = False
 
+GITHUB_CLIENT_ID = os.getenv('GITHUB_CLIENT_ID')
```

Comment:
Same here: Changes to lib/ folder shouldn't be part of this PR, right?

Reply:
Should we remove all the changes in auth-middleware then?

Reply:
Yes, you probably want to revert to the previous one.